### ISSUE TITLE
🔥 Elementor Pro v3.4 Beta 5 Release 🔥

### ISSUE THREAD

**Issue by avivu** [elementor/elementor#15947]
===========
# [Beta 5 UPDATED - PLEASE READ BELOW!](https://github.com/elementor/elementor/issues/15947#issuecomment-907771393)
===========

Hi Elementor Beta testers 👋 
 
We are incredibly excited to introduce our new beta version, Elementor Pro v3.4.
This version is all about expanding e-commerce capabilities, improving design options and updates to existing widgets.

In this version you’ll find two of the most exciting updates, including the revamped WooCommerce Menu Cart Widget, and the new ability to use product content on any page (including title, image, description, price, and more). Also, this version focuses on the addition of Load More functionality, Form widget improvements, as well as adding Vimeo to the Video Playlist widget. We’ve also made major performance improvements and reduced JS files size on each page load by up to 101KB.

Try out Elementor Pro v3.4 Beta and let us know what you think about our newest additions, including:

- **WooCommerce revamped Menu Cart widget**
- **WooCommerce Dynamic Tags**
- **AJAX Load More**
- **Form widget improvements**
- **Vimeo to Video Playlist widget**

Before you test, please make sure you are using:
- Elementor v3.4.2
- Elementor Pro v3.4.0 beta 5

# Introducing Revamped Menu Cart Widget: Mini Cart, AJAX reload and enhanced styling experience
![Menu Cart](https://user-images.githubusercontent.com/76733085/128906131-22e45ae2-0cd3-4126-b241-30e09236f757.png)
 
When it comes to conversions, the menu cart is one of the most influential parts of any eCommerce website, which is why we decided to expand and improve the menu cart widget. Now you’ll be able to style and customize your WooCommerce cart, as well as add a Mini Cart layout to the cart widget.

### New functionality for better store experience

The new menu cart widget offers the possibility to open the cart automatically after the item is added. This allows users to follow a cart summary at all times, enabling a smooth shopping experience. Also, we added an AJAX shopping cart functionality that can prevent pages from loading again after removing a product from the cart. Another new feature you can try out is opening the menu cart by click or by hover. 

### New cart type: Mini Cart

Under the Menu Cart widget you’ll find the new mini cart layout.
The Mini Cart is a floating cart that can be added anywhere on your WooCommerce website.  This layout creates a better user shopping experience, allowing easy access to the cart, the option to remove products and keep track of the subtotal and all in a more intuitive and nonintrusive way.

### New layout and style options

Now you can change and adjust your cart’s style and layout without adding a single line of code. Change the cart’s background color, choose the typography you want for the subtotal section, edit the icon’s positioning when the cart closes, and much more.

#### Here’s how it works:

1. Make sure the WooCommerce plugin is installed and activated. You must also have several products on your website
2. Create a new page, edit it with Elementor and drag in the Menu Cart widget
3. Choose between two cart types: Side Cart and Mini Cart
4. Explore the new cart layouts and controls and make sure everything works as expected
5. Check the new cart functionality under the Additional Options section, and make sure the AJAX option is working
6. Go to the Style tab and play around with the Menu Cart options, such as a cart, products, button, and custom message
7. Check an existing Menu Cart and make sure everything is working correctly with the new changes
8. Make sure everything works as expected in the Editor and on the Frontend

# Add WooCommerce Product Dynamic Tags To Anywhere On Your Site
![Dynamic tags](https://user-images.githubusercontent.com/76733085/128907331-71b6e945-385a-44c6-b9f0-edcabf3940e0.png)

 
WooCommerce Dynamic Tags refer to the content that can be drawn from the product and changed dynamically according to the product it’s on. Product dynamic tags range from price, rating, and sale status, to description, SKU, stock, terms, and title. You now have the ability to include these dynamic tags anywhere the product is presented throughout the website pages (Hero image, Homepage, Landing page, and more).

### Place your product anywhere you want 

Draw content from any product dynamic tag you want and place it on a new website page to reflect the product’s information. You can have the product presented, for example, on your home page, inside a hero image, or even on a dedicated landing page. 

### Dynamic changes

This new dynamic possibility means that the changes you make to the product’s values will reflect everywhere else the product is present. Once you change the price, description, etc. of the product, it will change everywhere else, whether it’s the product page itself or somewhere completely different.

#### Here’s how it works:

1. Make sure the WooCommerce plugin is installed and activated. You must also have several products on your website
2. Create a new page (not a Single Product template) and edit it using Elementor
3. Drag the Heading widget to the page
4. Go to the Title Dynamic Tags and choose the WooCommerce product title
5. Open the Dynamic Tags settings and search for a product
6. Make sure the Heading title changes to the one that you selected 
7. Repeat these actions with other WooCommerce dynamic tags and in different widgets
8. Check the WooCommerce dynamic product image by choosing a dynamic tag under the Image widget or a section background
9. Make sure everything works as expected in the Editor and on the Frontend

# Endless Scrolling With AJAX Load More Functionality
![Load More Options](https://user-images.githubusercontent.com/76733085/128907678-efc5e7a5-18e5-4fac-9e08-ca66906ef82d.png)

 
Showcasing content on your website has become more necessary than ever. However, there is a point where too much content and not enough space causes users to stop scrolling and leave your website. AJAX Load More is the ultimate design technique to display large amounts of content in an intuitive and appealing way. Now you can AJAX load more posts and archives.

### Infinite scroll or Load On Click 

In this version you’ll find two new pagination types to load more content - either by loading on click or enabling infinite scroll. Infinite scroll is a web design technique that loads content seamlessly as the user scrolls down a page. Content is automatically and continuously loaded at the bottom of the page, eliminating the need for pagination. While in some cases infinite load will be the better alternative, in some cases it will be best to load more content by clicking on a button to allow better user control.
Now you’ll be able to decide what best fits your website’s needs and adjust its style accordingly.

### Load More to Get More

Some benefits of not presenting all of your content at once includes:
- Increased Average Pageview Time - engage users with content they may never have uncovered otherwise.
- Lower Bounce Rate - remove the need for transition between pages and allow users to access content on the same page.
- Better User Experience - provide users with more content exposure, fewer clicks, and a native mobile experience while browsing through content.

#### Here’s how it works:

1. **Before you test!** Please make sure that the Improved Asset Loading experiment is set to “Active” for the Load More functionality to work perfectly. This limitation will be resolved in a later beta
2. Create a new page, edit it using Elementor and drag the Posts widget
3. Choose between two new pagination types: Load on Click and Infinite Scroll
4. Play around with the controls and choose a spinner, button option, and No more posts options. Check them with both of the new types
5. Go to the Style tab and play around with the pagination style options, such as button, spinner, and typography controls
6. Make sure everything works as expected in the Editor and on the Frontend
7. Please follow the steps above and also check the Posts Archive widget.

# Updating and Improving Existing Widgets
![Widgets Improvements - Forms + Vimeo](https://user-images.githubusercontent.com/76733085/128908152-4e1934f2-0793-4284-b082-757186ca0825.png)

 
We continually work on improving our Editor. As part of this effort we keep updating, adjusting and boosting the existing widgets and features you already have in use.
In the version we improved two widgets:

### Form Improvements 

- MailChimp Integration:
   - Made user groups loading more efficiently 
   - More informative error messages for administrators
   - New tweak: Double opt-in works properly
   - Adding tags do not remove the existing ones

- Resolved other bugs including Missing field IDs that caused some errors and no longer do

### Video Playlist

- Added Vimeo support to the video playlist widget

### Performance: Save up to ~100KB on each page load by dropping support for outdated browsers
Elementor was introduced 5 years ago, in the days when Internet Explorer and other alternatives were in the market and needed extensive support and duplication of code to interpret the modern code it was created by. Nowadays, web browsers are capable of [interpreting this modern language](https://caniuse.com/?search=ECMAScript%202015%20(ES6)) natively, without the need of additional interpretation.

Elementor [dropped support from IE11](https://developers.elementor.com/elementor-dropping-support-ie/) in version 3.0. In this version and the [corresponding Elementor core version](https://github.com/elementor/elementor/issues/15797), we optimized the transpilation process by dropping the support for old browsers such as IE11.

-----

## More Improvements

- Tweak: Added new Mini Cart layout type in Menu Cart widget (#11312)
- Tweak: Added styling options for Cart, Products, and Cart buttons in Menu Cart widget (#14952)
- Tweak: Added the ability to open cart automatically via Ajax when an item is added in Menu Cart widget (#14119)
- Tweak: Added the ability to remove an item from cart without reloading the page with an Ajax request (#9531, #10875, #10875, #11309)
- Tweak: Added Load More button functionality to Posts and Archive Posts widgets (#12126, #1284, #14557)
- Tweak: Added Vimeo support to the Video Playlist widget (#15319)
- Tweak: Improved asset loading performance by serving lighter JS files (#8572)
- Tweak: Added query string to the URL only after first video is played in Video Playlist widget (#15348)
- Tweak: Added various layout options with more responsive controls and cart structure options in Menu Cart widget
- Tweak: Added the option to open the menu cart by click or hover in Menu Cart widget
- Tweak: Added the ability to choose a specific product in WooCommerce Dynamic tags
- Tweak: Removed auto-scroll to the widget location when arriving from external URL in Video Playlist widget
- Tweak: Removed the Video Playlist widget from the Experiments list
- Tweak: Added descriptive error messages for MailChimp action after submit and on form submit in Forms widget
- Tweak: Added tooltip trigger None and Hover for link in Hotspot widget
- Tweak: Added responsive controls to Offset and Effects Offset in Sticky options
- Tweak: Added responsive control to Alignment in Testimonial Carousel widget
- Tweak: Adjusted Motion FX module to support future feature
- Fix: Products don't appear on cart while editing in Menu Cart widget (#15451)
- Fix: Videos always start muted in the Video Playlist widget (#15292)
- Fix: Unnecessary spacing if submenu indicator is set to None in Nav Menu widget (#15365)
- Fix: Mailchimp double opt-in feature doesn't work in Forms widget
- Fix: Fetching MailChimp groups field blocks the loading of the rest of the fields in the Form widget
- Fix: Missing field IDs causes forms not to be sent in Forms widget
- Fix: Clicking videos from the items list in edit mode doesn't initiate videos properly in Video Playlist widget
- Fix: Full Content Skin is not working properly when inserted twice on the same page in Posts widget
- Fix: Avoid Duplicates option doesn't exclude manual selections in Posts widget
- Fix: Submenu indicator alignment issue in Nav menu widget
- Fix: Query control deprecated message appears when debug mode is defined
- Fix: Tweet Button icon incorrect color and size when the icon is rendered as SVG in Blockquote widget
- Deprecated: See all deprecations to this version in our [Developers Deprecations Post](https://developers.elementor.com/v3-4-planned-deprecations/)

-----

## Make An Impact: Share Feedback Before Features Are Released
We’re always looking to get to know Elementor customers on a deeper level. If you’re interested in meeting with our team and sharing your insights, feedback, and feature requests you’re welcome to sign up for some quality time with us [here](https://uxpractices.elementor.cloud/joinresearch). Or download [Elementor’s Dev Edition](https://wordpress.org/plugins/elementor-beta/). Talk soon!

----

## How to Install the Beta Version
If you still haven't enabled beta testing in Elementor, please follow these steps:

- Go to Elementor > Tools > Versions
- Enable the beta tester feature
- Go to Dashboard > Updates and update to the latest beta version.
- This will get you set up to test the new Elementor Pro 3.4 Beta 5.

----

## Notes

For bugs and remarks, please reply with a comment on this issue here on GitHub.

**Please keep this related to the current beta only! For any other issue or feature request, feel free to open a new issue.**

Designated time for release: `August 31, 2021 (Estimated)`

Many thanks for your support and help!

Cheers,
_Elementor Team_

**Comment by fabianenzensperger** [comment]
Hi @avivu  and @shilo-ey ,

that's all very nice, but where are the individual icons? I'm not a programmer, so it is very cumbersome for me to change the icon using CSS. That was actually my only problem with this widget. All of these given icons rarely or never fit into the individual design language of a website.

<img width="301" alt="Bildschirmfoto 2021-08-10 um 21 08 42" src="https://user-images.githubusercontent.com/57997262/128921157-b20bfe9b-3a81-40b7-829b-ac8bf7f0ead8.png">


You should be able to add 1000% of your own symbol. Or at least choose from the Font Awesome library.

Please add this later in this beta. Thank you!

See you soon.

**Comment by 79ho3ein** [comment]
hello
I totally agree with @fabianenzensperger. I hope this issue is resolved in the next beta. Because being limited to a few icons does not feel good. While there is no limited use of icons in any other part of the elementor.
Please fix this if possible. The shopping cart menu widget requires the use of custom icons (svg and ...).

Finally, thanks for this wonderful update

**Comment by avivu** [comment]
@fabianenzensperger and @79ho3ein,
Custom icons are not part of this release, but we are working on adding them to a future release. You will find other great style and layout customization options that will help you create a cohesive website design.

Thank you for the feedback!

> Hi @avivu and @shilo-ey ,
> 
> that's all very nice, but where are the individual icons? I'm not a programmer, so it is very cumbersome for me to change the icon using CSS. That was actually my only problem with this widget. All of these given icons rarely or never fit into the individual design language of a website.
> 
> <img alt="Bildschirmfoto 2021-08-10 um 21 08 42" width="301" src="https://user-images.githubusercontent.com/57997262/128921157-b20bfe9b-3a81-40b7-829b-ac8bf7f0ead8.png">
> 
> You should be able to add 1000% of your own symbol. Or at least choose from the Font Awesome library.
> 
> Please add this later in this beta. Thank you!
> 
> See you soon.


@paauserrat 

In this Menu Cart addition, we added the option to style the "No Products In The Cart".
If you are talking about general WooCommerce notices, we are working on that.


> We should be able to edit the style and content of the woocommerce messages using elementor pro.


@mcrowell 
Currently, these are the controls for the Video Playlist widget.
Please open a new [Feature Request](https://github.com/pojome/elementor/issues/new?template=Feature_request.md) so we could deal with it more efficiently. 



> With the new Video Playlist, will we have the same Video Options as the Video widget?
> 
> It looks like most of the player options are missing:
> ![video-options](https://user-images.githubusercontent.com/7768009/129084490-6d4eafee-99e7-4faf-b1a9-3463ac2e80ff.jpg)

**Thanks again to everyone.**

**Comment by zeinnicholas** [comment]
> @fabianenzensperger and @79ho3ein,
> Custom icons are not part of this release, but we are working on adding them to a future release. You will find other great style and layout customization options that will help you create a cohesive website design.
> 
> Thank you for the feedback!

Hello @avivu, I agree with everyone else. Why they are not included with this release? Surely it would be as simple as changing controls, wouldn't it? In fact, looking at the code the implementation of the current icons is more complicated than using the Icon control... I don't get it. Is Elementor Pro supposed to be used as a locked-in limited theme or is Elementor Pro a professional design tool? This is as important as making any other design changes in this extremely important element of any e-commerce store. Why leave it for later (perhaps in three or two more years from now?) when it can be included in this release?

The same for the X "icon" used to remove items from the cart. Are we really going to have to accept this misaligned and invisible attempt of an icon? This is really bad for accessibility.
![image](https://user-images.githubusercontent.com/26891676/129195625-5f0e56ff-442c-4284-a347-bdc9e1aafe53.png)

I really don't get why every single release makes it such a struggle for you to be in line with us. Every single time you manage to get us excited, frightened, and disappointed at the same time.

At this point, I don't think it makes any difference but I'll try it anyway:
If you are going to release a Pro feature, either give us Pro controls or release it with core, heck, or even don't release it at all! We are tired of half-baked features!

I know this is really harsh, but how can I defend Elementor from all the negativity and complaints if you keep giving people reasons to complain? I'm not demanding you to fulfill all of our wishes, but I feel like my time spent giving feedback about things that should be improved was and still is being wasted... This is really frustrating!

You seriously need to invest in communication. You are completely out of touch with your customers!

And before you think this is related only to this release, I'm going to make it clear that this has been happening since Elementor 2.7. In every. Single. Release.

Thank you for this amazing tool you've created. I'll be always grateful. But I'm seriously reconsidering if I'll keep using it.

**Comment by promocot** [comment]
who updated the pro version after the last release, the optimization drops very much ... the plugin is developing but something is not in the right direction ... I'm starting to think about switching after 2 years of successful work, I'm disappointed with how everything is

**Comment by sanjupanikkar** [comment]
The menu cart button is not inheriting button style from Site Settings. 

The style applies correctly when I go into the Site Settings in the editor, but it resets to an unstyled button on the actual page. Inspecting the button in the console shows undefined variables for its colours and borders.
___
 The Posts widget is displaying a big SVG loading spinner. It appears even when the new loading options are not selected.

**Comment by jandrabek** [comment]
WooCommerce Dynamic Tags
I always wanted to display content of the product based on dynamic tags right from popup. Something like Quick View product plugin. Does this update allow me to do this?

**Comment by fabianenzensperger** [comment]
> > @fabianenzensperger and @79ho3ein,
> > Custom icons are not part of this release, but we are working on adding them to a future release. You will find other great style and layout customization options that will help you create a cohesive website design.
> > Thank you for the feedback!
> 
> Hello @avivu, I agree with everyone else. Why they are not included with this release? Surely it would be as simple as changing controls, wouldn't it? In fact, looking at the code the implementation of the current icons is more complicated than using the Icon control... I don't get it. Is Elementor Pro supposed to be used as a locked-in limited theme or is Elementor Pro a professional design tool? This is as important as making any other design changes in this extremely important element of any e-commerce store. Why leave it for later (perhaps in three or two more years from now?) when it can be included in this release?
> 
> The same for the X "icon" used to remove items from the cart. Are we really going to have to accept this misaligned and invisible attempt of an icon? This is really bad for accessibility.
> ![image](https://user-images.githubusercontent.com/26891676/129195625-5f0e56ff-442c-4284-a347-bdc9e1aafe53.png)
> 
> I really don't get why every single release makes it such a struggle for you to be in line with us. Every single time you manage to get us excited, frightened, and disappointed at the same time.
> 
> At this point, I don't think it makes any difference but I'll try it anyway:
> If you are going to release a Pro feature, either give us Pro controls or release it with core, heck, or even don't release it at all! We are tired of half-baked features!
> 
> I know this is really harsh, but how can I defend Elementor from all the negativity and complaints if you keep giving people reasons to complain? I'm not demanding you to fulfill all of our wishes, but I feel like my time spent giving feedback about things that should be improved was and still is being wasted... This is really frustrating!
> 
> You seriously need to invest in communication. You are completely out of touch with your customers!
> 
> And before you think this is related only to this release, I'm going to make it clear that this has been happening since Elementor 2.7. In every. Single. Release.
> 
> Thank you for this amazing tool you've created. I'll be always grateful. But I'm seriously reconsidering if I'll keep using it.


Yeah, I'm just annoyed now. Whenever you think, "oh yes, cool, now a widget that is long overdue is being equipped with new functions", you will be disappointed again because such extremely important functions have not been considered. Or much too late? Why not just revise a widget in a sensible way? As you said, we're not at "Make a wish" here, but such essential functions should be possible. But yes, maybe in 1 to 2 years we will finally be able to completely tune the icon ourselves. Wow!

**Comment by black-eye** [comment]
> I really don't get why every single release makes it such a struggle for you to be in line with us. Every single time you manage to get us excited, frightened, and disappointed at the same time.
> 
> .....
> 
> I know this is really harsh, but how can I defend Elementor from all the negativity and complaints if you keep giving people reasons to complain? I'm not demanding you to fulfill all of our wishes, but I feel like my time spent giving feedback about things that should be improved was and still is being wasted... This is really frustrating!
> 
> You seriously need to invest in communication. You are completely out of touch with your customers!
> 
> And before you think this is related only to this release, I'm going to make it clear that this has been happening since Elementor 2.7. In every. Single. Release.
> 
> Thank you for this amazing tool you've created. I'll be always grateful. But I'm seriously reconsidering if I'll keep using it.

I cannot agree more  - with every single update you manage to get me excited and disappointed at the same time. Please wake up and start to listen to your core community and common sense! Every single update you get tons of reasonable feed back, which is not heard out. Why we have to start an endless battle to convice you for every little tweak everybody else considers no-brainer (breakpoint order in the responsive tab to name just one).

What else you need to hear or see, when even the volunteer moderator of your own official FB group begs you to improve your communication and asks you to listen to your user base and community!

**Comment by whipster-loaded** [comment]
Would like to comment on the new dynamic woocommerce tags.

When I (typically) build single product pages, I prefer to use column BG / or image widget for displaying a product image.  I do not use the woo product images widget due to it's magnifier aspect or lightbox functionality.  I just want to display the image statically.  

To do this I would simply choose the 'product image' tag.  

Now when I do this, it of course wants me to select a specific product to display.  So instead, I have to choose the 'Featured Image' tag to display the current product.

Is there a way for Elementor to add to the new dynamic tag, a dynamic query 'This' or 'Current Product' etc.  without the need for a type in field?  

With the update I would have to go into each product template on each site I've designed in my traditional way, and reset the image tag to select 'featured image'.

I think you should set the new tag to default to current product id, then have the option to choose a different product in the type field.  This would help me alot, not sure if anyone else designed the way I described at the beginning of the post.  

Please conisder this method.
THX guys, love the new mini-cart functionality btw, been waiting for that since forever :)
-W

**Comment by MichaelDarko** [comment]
Agree, Elementor just can't get rid of this Wix / Squarespace mentality - "Here are some widgets, here are some predefined options and that should be enough." This should not be happening at this point.

### **Few examples:**

**Clumn gap** - No gap / Narrow / Extended / Wide, Wider

**Form field input size** - Extra small / Small, Medium / Large, Extra large

**Buttons** - Extra small / Small, Medium / Large / Extra large

**Menu cart icon** - Cart / Basket / Bag / ...

**Post item box shadow** - On / Off (no styling, just a random box shadow)

**Social icons shape** - Rounded / Square / Circle

What are designers supposed to do with this?

You are still making Gutenberg with more styling options. You're not making a professional tool that gives us the freedom to design anything we want.

You keep adding widgets and options, but you still haven't made a framework of versatile basic building blocks.

Elementor should essentially be a layer of abstraction above code, a visual UI that outputs a website. Like Figma for making websites. In design tools in general, you have a few geometric shapes, colors, typography and some other options. Simple building blocks and you can build the most complex designs with it. Pre-built blocks can be a bonus, but it's not what matters in a design framework. 

In Elementor, you have a massive number of widgets and yet very limited design freedom. It was understandable in the beginning, it was the shortest route to something that does the job, but why is there no sign of changing direction when Gutenberg is going in the same direction?

You should not be better than Gutenberg, you should be something different that Gutenberg. You should be Webflow for WordPress, Unreal Engine for Wordpress, Figma for Wordpress.

If you look at the most requested features here on Github, it's essentially people telling you "Stop giving us new widgets, give us the freedom to design, give us the basics."

It would be nice to see a gradual shift form a widget assembling tool to a design tool, but version after version it's just the same old approach. The longer you leave it, the harder it will be to shift.

Elementor is phenomenal at executing stuff, nobody is able to compete on this front, if you could combine that with solid first principles thinking, you would be unstoppable in the next decade.

**Comment by sanjupanikkar** [comment]
Another issue: the invisible container for the menu cart is blocking the the mini controls for sections, making them difficult to click.
___
> Would like to comment on the new dynamic woocommerce tags.
> 
> When I (typically) build single product pages, I prefer to use column BG / or image widget for displaying a product image. I do not use the woo product images widget due to it's magnifier aspect or lightbox functionality. I just want to display the image statically.
> 
> To do this I would simply choose the 'product image' tag.
> 
> Now when I do this, it of course wants me to select a specific product to display. So instead, I have to choose the 'Featured Image' tag to display the current product.
> 
> Is there a way for Elementor to add to the new dynamic tag, a dynamic query 'This' or 'Current Product' etc. without the need for a type in field?
> 
> With the update I would have to go into each product template on each site I've designed in my traditional way, and reset the image tag to select 'featured image'.
> 
> I think you should set the new tag to default to current product id, then have the option to choose a different product in the type field. This would help me alot, not sure if anyone else designed the way I described at the beginning of the post.
> 
> Please conisder this method.
> THX guys, love the new mini-cart functionality btw, been waiting for that since forever :)
> -W
> 
> EDIT***
> 
> Seems none of the dynamic tags work anymore on my single product pages...I'm confused...Why aren't the tags defaulting to the current product ID for all the Woo widgets.

Yes, this is a HUGE omission for an update that's about adding WooCommerce tags. There needs to be a "current product" option so that building WooCommerce pages can be streamlined into the general purpose widgets.

However, this would duplicate the functionality of many of the WooCommerce specific product widgets.

**Comment by whipster-loaded** [comment]
> Yes, this is a HUGE omission for an update that's about adding WooCommerce tags. There needs to be a "current product" option so that building WooCommerce pages can be streamlined into the general purpose widgets.

> However, this would duplicate the functionality of many of the WooCommerce specific product widgets.

Agree.  The current product ID either needs to be built in as the default selector of the dynamic tag, or in effect they would have to duplicate the general purpose 'woo single product' widgets into 'custom woo single product' widgets in effect.  Something like how they utilize the 'add to cart' and the 'custom add to cart' widgets.

**Comment by datverse** [comment]
Hi, can you add 1 more option "simple" layout for Menu Cart Widget (cart type): "Only Button" Style. With this style / layout Menu Cart Widget have Link to "View Cart", quantity product, total price. No Mini Popup Cart, No Side Cart (cart types: Side Cart and Mini Cart). We can style the Button, quantity product, total price.
Thanks

**Comment by Brentnauer** [comment]
> Agree, Elementor just can't get rid of this Wix / Squarespace mentality - "Here are some widgets, here are some predefined options and that should be enough." This should not be happening at this point.
> 
> ### **Few examples:**
> **Clumn gap** - No gap / Narrow / Extended / Wide / Wider
> 
> **Form field input size** - Extra small / Small / Medium / Large / Extra large
> 
> **Buttons** - Extra small / Small / Medium / Large / Extra large
> 
> **Menu cart icon** - Cart / Basket / Bag / ...
> 
> **Post item box shadow** - On / Off (no styling, just a random box shadow)
> 
> **Social icons shape** - Rounded / Square / Circle
> 
> What are designers supposed to do with this?
> 
> You are still making Gutenberg with more styling options. You're not making a professional tool that gives us the freedom to design anything we want.
> 
> You keep adding widgets and options, but you still haven't made a framework of versatile basic building blocks.
> 
> Elementor should essentially be a layer of abstraction above code, a visual UI that outputs a website. Like Figma for making websites. In design tools in general, you have a few geometric shapes, colors, typography and some other options. Simple building blocks and you can build the most complex designs with it. Pre-built blocks can be a bonus, but it's not what matters in a design framework.
> 
> In Elementor, you have a massive number of widgets and yet very limited design freedom. It was understandable in the beginning, it was the shortest route to something that does the job, but why is there no sign of changing direction when Gutenberg is going in the same direction?
> 
> You should not be better than Gutenberg, you should be something different that Gutenberg. You should be Webflow for WordPress, Unreal Engine for Wordpress, Figma for Wordpress.
> 
> If you look at the most requested features here on Github, it's essentially people telling you "Stop giving us new widgets, give us the freedom to design, give us the basics."
> 
> It would be nice to see a gradual shift form a widget assembling tool to a design tool, but version after version it's just the same old approach. The longer you leave it, the harder it will be to shift.
> 
> Elementor is phenomenal at executing stuff, nobody is able to compete on this front, if you could combine that with solid first principles thinking, you would be unstoppable in the next decade.

QFE. Agree 100%, feeling more and more boxed in as time goes on rather than empowered to create whatever.

**Comment by edknewseed** [comment]
I am sure all of you have a beta site set up for testing Elementor Beta.  I can't say that I experience the same issues but I have been hear about the 3.4 beta release for quite some time know and although I like some of the improvements, I thought that this 3.4 version will winding down to just fixing bug issues including the new features because what's bugging me is that when I go to do the update for the standard beta version I noticed the version is different if I go to Dashboard -> Updates versus updating it via Plugins.   According to the Dashboard update method, it shows to be version 3.4.0-dev13 and the new version it wants to update is 3.5.0-dev6.   

So my question is this:  Are we still testing the 3.4 beta in which when ready it will be release to the public as 3.4?  Or are they skipping 3.4 and moved onto 3.5?

Note: I decided not to update to 3.5 just yet. Just wanted to know what's going on about the versions.

Thank you.

**Comment by d19dotca** [comment]
> I am sure all of you have a beta site set up for testing Elementor Beta.  I can't say that I experience the same issues but I haven't put much effort into testing lately but what I have been hearing about is the 3.4 beta release for quite some time now and although I like some of the improvements, I thought that this 3.4 version will be winding down to just fixing bug issues including the new features and then release.

I think you may be confusing Elementor Pro 3.4 beta (which just started) with Elementor (core) 3.4 (which I believe is nearly done). These are different products and different beta periods.

**Comment by black-eye** [comment]
> 
> 
> > I am sure all of you have a beta site set up for testing Elementor Beta.  I can't say that I experience the same issues but I haven't put much effort into testing lately but what I have been hearing about is the 3.4 beta release for quite some time now and although I like some of the improvements, I thought that this 3.4 version will be winding down to just fixing bug issues including the new features and then release.
> 
> I think you may be confusing Elementor Pro 3.4 beta (which just started) with Elementor (core) 3.4 beta (which I believe is nearly done).
> 
> These are different products and different beta periods.

I don't think so. Most probably @edknewseed has got installed Developers Edition which goes beyond the last beta. This Developers Edition should have brought more clarity to beta testing, but unfortunately it brought just another level of confusion.  :-(

**Comment by projektorius96** [comment]
Per basis of the bug #15451 issues, I do consider it's still on the way to be fixed, please correct me If I am wrong. Thank you.

**Comment by jazir5** [comment]
@DorShahar @shilo-ey @udidol 

Why aren't you guys hearing us about feedback? It seems like you ignore your users requesting stuff in beta threads over and over. I agree with the other commenters here. Please respond and implement the controls we have been asking for for years. Icon control of the minicart widget is essential. I'm using powerpack addons and they have this feature for the mini-cart. 

The stock elementor widget should have just as many or more controls. I see no reason to use this widget over a third party addon pack when it is subpar compared to what they offer. If you are going to implement a feature into elementor core or pro, it should be better than the addon pack competitors. Otherwise, what is even the point of implementing these features?

**Comment by projektorius96** [comment]
@avivu with reference to the bug Menu cart widget do not render properly #15451 issued , finally fixed ! God bless you fellas !

**Comment by zeinnicholas** [comment]
> Would like to comment on the new dynamic woocommerce tags.
> 
> When I (typically) build single product pages, I prefer to use column BG / or image widget for displaying a product image. I do not use the woo product images widget due to it's magnifier aspect or lightbox functionality. I just want to display the image statically.
> 
> To do this I would simply choose the 'product image' tag.
> 
> Now when I do this, it of course wants me to select a specific product to display. So instead, I have to choose the 'Featured Image' tag to display the current product.
> 
> Is there a way for Elementor to add to the new dynamic tag, a dynamic query 'This' or 'Current Product' etc. without the need for a type in field?
> 
> With the update I would have to go into each product template on each site I've designed in my traditional way, and reset the image tag to select 'featured image'.
> 
> I think you should set the new tag to default to current product id, then have the option to choose a different product in the type field. This would help me alot, not sure if anyone else designed the way I described at the beginning of the post.
> 
> Please conisder this method.
> THX guys, love the new mini-cart functionality btw, been waiting for that since forever :)
> -W
> 
> EDIT***
> 
> Seems none of the dynamic tags work anymore on my single product pages...I'm confused...Why aren't the tags defaulting to the current product ID for all the Woo widgets.

I completely agree with this too. This is important stuff. If you are going to change how it works, then make it right the first time around.

**Comment by parfilov** [comment]
**Add WooCommerce Product Dynamic Tags To Anywhere On Your Site** — very controversial solution
**1)** Too many clicks to choose product.
1 Title, 2 price, 3 image, 4 button — are minimum and for only one product. What if I have xN products?..

**2)** Ok, I placed the title, price, image (finger already tired) and I want to place button. No problem. But what should I choose in dynamic tags, WooCommerce > Product Link? No, it would be too simple. I should guess to use Internal URL option.

**3)** However, what if I want to use Add To Cart widget to let user to add this product to cart without going to single product. No problem. But if I click on it, it redirects me to the product page — awesome! Where did you say there was ajax adding in the Menu Cart widget?.. Why it doesn't work with this widget?

**Comment by parfilov** [comment]
@zeinnicholas What am I wrong?

**Comment by avivu** [comment]
> We should be able to edit the style and content of the woocommerce messages using elementor pro.

Hello @paauserrat,
Regarding this release, you can style the Menu Cart content.
If you refer to the general WooCommerce notices, it's not part of this version, but we are working on that for a future release.



> The menu cart button is not inheriting button style from Site Settings.
> 
> The style applies correctly when I go into the Site Settings in the editor, but it resets to an unstyled button on the actual page. Inspecting the button in the console shows undefined variables for its colours and borders.
> 
> The Posts widget is displaying a big SVG loading spinner. It appears even when the new loading options are not selected.

@sanjupanikkar 
Can you please add more details about the Site Settings issue? Maybe add a quick video with the problem?

SVG spinner - We are aware of this issue and will fix it in the next beta.
Thank you!




> WooCommerce Dynamic Tags
> I always wanted to display content of the product based on dynamic tags right from popup. Something like Quick View product plugin. Does this update allow me to do this?

@jandrabek 
This addition to the dynamic tags allows you to choose a specific product to display anywhere you want, but it's not a quick view option.




> Would like to comment on the new dynamic woocommerce tags.
> 
> When I (typically) build single product pages, I prefer to use column BG / or image widget for displaying a product image. I do not use the woo product images widget due to it's magnifier aspect or lightbox functionality. I just want to display the image statically.
> 
> To do this I would simply choose the 'product image' tag.
> 
> Now when I do this, it of course wants me to select a specific product to display. So instead, I have to choose the 'Featured Image' tag to display the current product.
> 
> Is there a way for Elementor to add to the new dynamic tag, a dynamic query 'This' or 'Current Product' etc. without the need for a type in field?
> 
> With the update I would have to go into each product template on each site I've designed in my traditional way, and reset the image tag to select 'featured image'.
> 
> I think you should set the new tag to default to current product id, then have the option to choose a different product in the type field. This would help me alot, not sure if anyone else designed the way I described at the beginning of the post.
> 
> Please conisder this method.
> THX guys, love the new mini-cart functionality btw, been waiting for that since forever :)
> -W
> 
> EDIT***
> 
> Seems none of the dynamic tags work anymore on my single product pages...I'm confused...Why aren't the tags defaulting to the current product ID for all the Woo widgets.

@whipster-loaded @sanjupanikkar 
We will address this issue and fix it on the next beta, and it will work as it always worked.
Thank you!



> @avivu with reference to the bug Menu cart widget do not render properly #15451 issued , finally fixed ! God bless you fellas !

@projektorius96 
Great! I'm happy to hear that :)

**Comment by jazir5** [comment]
@avivu 

Why did you ignoring everyone asking you to reconsider things beta users are complaining about. Can you please issue a response?

**Comment by avivu** [comment]
> @avivu
> 
> Why did you ignoring everyone asking you to reconsider things beta users are complaining about. Can you please issue a response?

@jazir5 
We are not ignoring. A response to that will be later on.

**Comment by avivu** [comment]
Hello everyone,

I would like to thank all of you for the comments. It’s important for us to hear your thoughts and to address them. 

When it comes to thinking about a new feature or a widget release, a lot of elements come into consideration. Making sure the new product will not change, update or create something that can potentially damage existing sites or harm website performance. Also taking into consideration market demand and necessity, technological progress and of course, user requests. In each release, the goal is to bring a solution that will best fit the user's needs. 


We are aware that custom SVG is an important feature for creating tailored-made eCommerce websites. At this point, the release contains updates to the Menu Cart widgets that give you more style and layout customization. Custom SVG is not forgotten and we are working to enable it. Changing the current icon controls may break existing sites, and we want to find a solution that will provide a seamless experience of this feature alongside maintaining stable websites and avoiding database modifications. We are determined to ship this feature, but we want to do it right.  


We are working on improving and extending the WooCommerce integration, and this new version is just the starting point. We will continuously release new updates and features that will help you create the best online stores you can.


Please keep this thread related to the current beta only.  Your comments are valued, and we are listening. If you’re interested in meeting with our team and sharing your insights, feedback, and feature requests, you’re welcome to sign up for some quality time with us [here](https://uxpractices.elementor.cloud/joinresearch/).

Thank you all!

**Comment by gem-design** [comment]
Dropping support for IE11 is fine, despite it having a non negligible over 5% market share. CSS size, JS size, Image Size, DOM size are not the real issue with performance. The time it takes elementor to render a dynamic design from a template is the issue. TTFB is the issue. 

101KB of JS can be cached, if not it's not that big for modern connection (I presume we're targeting modern devices since we are eliminating the non modern IE11), if not it can be loaded conditionally based on the user-agent. it can be enqueued in the footer...

Dynamic pages with enormous TTFB time can't be cached, can't be conditional, it's slow no matter the user's internet speed... it affects all users, it's not something that can be enqueued... Full page caching isn't an option for everyone.

Have you considered optimizing rendering when the template is saved, generate a PHP template when a user saves the design so that when it's time to render there is no going to the database and parsing JSON and dealing with conditions?... 

I love elementor, I can't and won't change it with any other builder... please love us back and help with performance for users taking elementor to the next level by building dynamic, feature rich, web apps on elementor.

**Comment by jazir5** [comment]
>101KB of JS can be cached, if not it's not that big for modern connection (I presume we're targeting modern devices since we are eliminating the non modern IE11), if not it can be loaded conditionally based on the user-agent. it can be enqueued in the footer...

Js size is very, very important for how fast the page loads. JS data takes significantly more time to render than css. These file size reductions from dropping IE support will be very essential for improving page load speed. Caching is not a good solution for this, you need to already have that implemented for the best performance. 

>Dynamic pages with enormous TTFB time can't be cached, can't be conditional, it's slow no matter the user's internet speed... it affects all users, it's not something that can be enqueued... Full page caching isn't an option for everyone.

Dynamic page actually do not have any effect on performance in my experience. An example of this is the single product template. My single product pages do not suffer any additional page load time just due to have dynamic content loaded. You need a caching solution, as well as a myriad of other optimizations to work correctly. Caching alone is not a good enough solution.

Css loading and javascript loading optimizations that the elementor team have implemented so far have resulted in massive performance improvements to the frontend. The asset loading experiment boosted my scores by 15 points on mobile lighthouse.

**Comment by gem-design** [comment]
> Js size is very, very important for how fast the page loads. JS data takes significantly more time to render than css. These file size reductions from dropping IE support will be very essential for improving page load speed. Caching is not a good solution for this, you need to already have that implemented for the best performance.
> 
I'm not saying it's not important, but I already provided multiple ways to improve it (the user-agent solutions is two lines of code and a closed and shut case), unlike pages with very high (3 Second+ TTFB) JS can be optimized and the user will see a lot of information and have the ability to interact with the page even before JS is loaded parsed and executed, I'm a nodejs developer and in my experience JS rendering time is only an issue when talking about a phone from 2014, On my elementor websites/webapps the issue not measured in lighthouse score, I can see the user waiting for multiple seconds for the page to show anything.
 
> Dynamic pages actually do not have any effect on performance in my experience. An example of this is the single product template. My single product pages do not suffer any additional page load time just due to have dynamic content loaded. You need a caching solution, as well as a myriad of other optimizations to work correctly. Caching alone is not a good enough solution.
> 

The issue isn't with a single product page or single blog post page, these pages are just loading the content, if the issue was there we could just use the theme version and edit the CSS and hook into theme/wordpress/woocommerce actions and filters...

I'm talking about dashboard with user generated editable content, archive pages with custom loops, filters, user / post conditions, with dynamic popups for interaction. user specific widgets and views, advanced user-friendly profile builders... it adds up quick and TTFB eats up to 80% of loading time.

> Css loading and javascript loading optimizations that the elementor team have implemented so far have resulted in massive performance improvements to the frontend. The asset loading experiment boosted my scores by 15 points on mobile lighthouse.

Before CSS loads, before JS loads, before images, before the page becomes interactive... The document needs to be loaded first. While the browser is waiting for the response it's showing a white blank page... unlike the 5 different ways to fix the 101KB of JS, there is no way around the long TTFB even after throwing a 4GB VPS with openlitespeed behind the site and even if the user is on fiber connection. 

PS: You got your performance issue optimized and fix, why are you denying other's performance issues?

**Comment by avivu** [comment]
Hey all,
We just released a new beta version, please update your version to **Pro v3.4.0 Beta 2**.

- Tweak: Added future compatibility to support better loading of eicons font
- Fix: Load More spinner icon is always presented when Font-Awesome Inline experiment is active in Posts widget
- Fix: Video icon size is not changing on Active state in Video Playlist widget
- Fix: Header icon color is not working in Table Of Content widget
- Fix: Posts are not wrapped with an article tag when using the Load More functionality in Posts and Post Archive widgets
- Fix: Legacy mode is missing the new code for Load More functionality in Posts and Post Archive widgets
- Fix: Video duration for Vimeo isn't fetched correctly in Video Playlist widget
- Fix: Price & Quantity ampersand sign printed as HTML in Menu Cart widget


**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/pojome/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thanks again to everyone!

**Comment by maurocarreraok** [comment]
Come one guys... Add custom icons to CART please.........

**Comment by DTR96** [comment]
How can i use this beta version?

**Comment by d19dotca** [comment]
> How can i use this beta version?

There's a section in the main post about how to install the beta version. Additionally there is documentation here: https://elementor.com/help/how-to-become-a-beta-tester/

**Comment by udidol** [comment]
@DTR96 
The beta doesn't exist anymore, the production version of 3.4.0 has been fully released. We've even released a patch version already, v3.4.1

**Comment by d19dotca** [comment]
I noticed that when using the Nav Menu in Elementor Pro with the "Font-Awesome Inline" experiment active, that the chevron icon for the submenu fails to change hover colour. 

How it should look like (when experiment is inactive):
<img width="294" alt="image" src="https://user-images.githubusercontent.com/8153796/130342497-b88ac148-fcda-4f48-b9e8-304753e85bd1.png">

How it looks when the experiment is active (bug):
<img width="287" alt="image" src="https://user-images.githubusercontent.com/8153796/130342515-b1114b64-dfdf-4951-9bf4-9d88aa083e2b.png">

**Comment by avivu** [comment]
> I noticed that when using the Nav Menu in Elementor Pro with the "Font-Awesome Inline" experiment active, that the chevron icon for the submenu fails to change hover colour.
> 
> How it should look like (when experiment is inactive):
> <img alt="image" width="294" src="https://user-images.githubusercontent.com/8153796/130342497-b88ac148-fcda-4f48-b9e8-304753e85bd1.png">
> 
> How it looks when the experiment is active (bug):
> <img alt="image" width="287" src="https://user-images.githubusercontent.com/8153796/130342515-b1114b64-dfdf-4951-9bf4-9d88aa083e2b.png">

Hello @d19dotca,
What Pro version are you using? We release a fix for that in Pro 3.3.6

**Comment by d19dotca** [comment]
> > I noticed that when using the Nav Menu in Elementor Pro with the "Font-Awesome Inline" experiment active, that the chevron icon for the submenu fails to change hover colour.
> 
> > 
> 
> > How it should look like (when experiment is inactive):
> 
> > <img alt="image" width="294" src="https://user-images.githubusercontent.com/8153796/130342497-b88ac148-fcda-4f48-b9e8-304753e85bd1.png">
> 
> > 
> 
> > How it looks when the experiment is active (bug):
> 
> > <img alt="image" width="287" src="https://user-images.githubusercontent.com/8153796/130342515-b1114b64-dfdf-4951-9bf4-9d88aa083e2b.png">
> 
> 
> 
> Hello @d19dotca,
> 
> What Pro version are you using? We release a fix for that in Pro 3.3.6

I just realized I'm on Pro 3.3.7. So I apologize as I thought I had enabled the beta on this site. However if the fix was in 3.3.6 then it's broken again in 3.3.7 I guess. I can open a new issue for this though since it isn't for the beta version. I'll try the beta on this same site later and see if it fixes that issue though too in case it's also affecting the beta.

**Comment by 79ho3ein** [comment]
Hi everyone🖐
Unfortunately, for some reason I could not use the beta version so I could review it ... But I want to take this opportunity to describe a very important bug that existed before this update. I do not know if this bug is in the new version or not! I hope dear developers will investigate this problem. this bug has occurred to several colleagues, and certainly to you. **Fixing this bug has no choice but to rebuild the site!**

**And the description of the bug ...**
Suppose we use another plugin to design a menu cart ,For example, I created my site menu cart with PowerPack plugin. 
At the same time, I added the elementor menu cart widget to the page and edited it a bit. After that, the old menu cart (Power pack plugin shopping cart menu) was completely changed and became the elementor menu cart.
**This was not the end of the problem!** After that, the elementor menu cart could no longer be edited. And had its original appearance. **This means that both the power pack menu cart widget and elementor were unusable.** 

Before I reported this bug, this problem occurred to a friend in the form of Woodmart thame.She added the elementor menu cart widget to her page for a few minutes, after which the Woodmart menu cart became the elementor menu cart and she could not make any changes and had to use her backup.

I apologize for the lengthy explanation. In any case, this is a very terrible bug, I tried to describe it in the best way. **Please friends if you experienced such a similar problem, share it.**
Finally, I have to say that I experienced this problem in the hello theme.

Thank you all

**Comment by jazir5** [comment]
> > > I noticed that when using the Nav Menu in Elementor Pro with the "Font-Awesome Inline" experiment active, that the chevron icon for the submenu fails to change hover colour.
> > 
> > 
> > > 
> > 
> > 
> > > How it should look like (when experiment is inactive):
> > 
> > 
> > > <img alt="image" width="294" src="https://user-images.githubusercontent.com/8153796/130342497-b88ac148-fcda-4f48-b9e8-304753e85bd1.png">
> > 
> > 
> > > 
> > 
> > 
> > > How it looks when the experiment is active (bug):
> > 
> > 
> > > <img alt="image" width="287" src="https://user-images.githubusercontent.com/8153796/130342515-b1114b64-dfdf-4951-9bf4-9d88aa083e2b.png">
> > 
> > 
> > Hello @d19dotca,
> > What Pro version are you using? We release a fix for that in Pro 3.3.6
> 
> I just realized I'm on Pro 3.3.7. So I apologize as I thought I had enabled the beta on this site. However if the fix was in 3.3.6 then it's broken again in 3.3.7 I guess. I can open a new issue for this though since it isn't for the beta version. I'll try the beta on this same site later and see if it fixes that issue though too in case it's also affecting the beta.

This is broken for me too, i'm on 3.3.7

**Comment by maurocarreraok** [comment]
Elementor...... You did it again. I'm so angry. This happened before and nothing changed.

Please, push a fix for ALL THE ISSUES caused by this latest release. Paddings, Margins, Background Positions on different breakpoints, dynamic colors and dynamic content, completely broken...

I'm getting tired of rushing and saying “sorry” to my clients...

**Comment by avivu** [comment]
Hey all,
We just released a new beta version, please update your version to **Pro v3.4.0 Beta 3.**

- Fix: WooCommerce Dynamic Tags are not dynamically in non WooCommerce widgets in the Single Product template
- Fix: Product Terms not retrieving specific product's taxonomy in WooCommerce Dynamic Tags
- Fix: Load more functionality is not working in Posts widget when inserted into Archive template
- Fix: Icons style conflicts when Font-Awesome Inline experiment is active in Video Playlist widget
- Fix: JS Error console message when using the Posts and Archive Posts widgets
- Fix: Elements loading issues due to conflicts between Motion FX module and future feature
- Fix: Menu Icon border hover color is not changing in Menu Cart widget
- Fix: Cart and Products borders appears when the border type is set to None in Menu Cart 
- Fix: Buttons padding are not working as expected in Menu Cart widget

**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/elementor/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/elementor/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thanks you!

**Comment by jazir5** [comment]
> Hey all,
> We just released a new beta version, please update your version to **Pro v3.4.0 Beta 3.**
> 
> * Fix: WooCommerce Dynamic Tags are not dynamically in non WooCommerce widgets in the Single Product template
> * Fix: Product Terms not retrieving specific product's taxonomy in WooCommerce Dynamic Tags
> * Fix: Load more functionality is not working in Posts widget when inserted into Archive template
> * Fix: Icons style conflicts when Font-Awesome Inline experiment is active in Video Playlist widget
> * Fix: JS Error console message when using the Posts and Archive Posts widgets
> * Fix: Elements loading issues due to conflicts between Motion FX module and future feature
> * Fix: Menu Icon border hover color is not changing in Menu Cart widget
> * Fix: Cart and Products borders appears when the border type is set to None in Menu Cart
> * Fix: Buttons padding are not working as expected in Menu Cart widget
> 
> **Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/elementor/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/elementor/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**
> 
> Thanks you!

@avivu 

Can you fix this bug for the 3.4 pro release?

https://github.com/elementor/elementor/issues/16136

**Comment by avivu** [comment]
Hey all,
We just released a new beta version, please update your version to **Pro v3.4.0 Beta 4.**

- Fix: Bullet markers are not visible on preview mode when Font Awesome Inline experiment is active in Table of Content widget
- Fix: UI Glitch when Font-Awesome Inline experiment is active in Price Table widget
- Fix: WooCommerce Price dynamic tag prints HTML in Animated Headline widget
- Fix: Error message appears if Motion FX is active in debug mode
- Fix: Submenu Indicator appears larger when Font Awesome Inline experiment is active in Nav Menu widget
- Deprecated: Remove all usages of `Elementor\Utils::get_create_new_post_url()`


**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/elementor/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/elementor/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thank you!

**Comment by 79ho3ein** [comment]
> Please comment on the svg inline experiment making the chevron icon invisible in the hamburger menu on mobile until tapped. It's completely unusable for me while this bug exists.

Unfortunately now elementor is drowning in bugs...

**Comment by Michaelgimii** [comment]
hi @avivu
Will we see the custom icon feature for the menu cart widget in 2021? Or in other years?

**Comment by avivu** [comment]
@maurocarreraok and @jazir5 
We will examine these issues and continue the conversation on the dedicated git issue.
Thank you!

> Please, comment on the bugs people are reporting.
> #16131
> #14672


> Please comment on the svg inline experiment making the chevron icon invisible in the hamburger menu on mobile until tapped. It's completely unusable for me while this bug exists.




@Michaelgimii 
A custom icon is not forgotten, and we are working to enable it. We want to find a solution that will fit various future features and work correctly on existing websites.


> hi @avivu
> Will we see the custom icon feature for the menu cart widget in 2021? Or in other years?

**Comment by avivu** [comment]
Hey all,
We just released a new beta version, please update your version to **Pro v3.4.0 Beta 5**.

- Tweak: Changed Rename Part title and button color in Theme Builder
- Fix: Part name is deleted when clicking on the "Change" Button without changing the name in Theme Builder

Thank you!

**Comment by d19dotca** [comment]
I see that Elementor Pro 3.4 is now being released formally, as a couple of my websites recognize the new update.
